pkg/gradle/lockfile/parse.go

```diff
@@ -0,0 +1,39 @@
+package lockfile
+
+import (
+	"bufio"
+	"strings"
+
+	dio "github.com/aquasecurity/go-dep-parser/pkg/io"
+	"github.com/aquasecurity/go-dep-parser/pkg/types"
+	"github.com/aquasecurity/go-dep-parser/pkg/utils"
+)
+
+type Parser struct{}
+
+func NewParser() types.Parser {
+	return &Parser{}
+}
+
+func (Parser) Parse(r dio.ReadSeekerAt) ([]types.Library, []types.Dependency, error) {
+	var libs []types.Library
+	scanner := bufio.NewScanner(r)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.HasPrefix(line, "#") { // skip comments
```

Comment:
I'd use something like [TrimSpace](https://pkg.go.dev/strings#TrimSpace) here, to skip next comments:
```yaml
 # comment
   # comment2
````

Reply:
or it doesn't matter here too...

Reply:
It is make sense.
I am not sure that is possible, but i added trim to be safe.